tests/test_watcher.py

```diff
@@ -0,0 +1,23 @@
+import os.path
+import functools
+
+from lektor import utils
+from lektor import watcher
+
+
+def test_is_interesting(env):
+    cache_dir = os.path.abspath(utils.get_cache_dir())
```

Comment:
Rather than calling `utils.get_cache_dir()`, I recommend using the [`tmpdir`](http://docs.pytest.org/en/latest/tmpdir.html) fixture from `pytest` to get a temporary directory. That way, you get to use the [py.path API](http://py.readthedocs.io/en/latest/path.html), to do nice things like this:
```python
def test_is_interesting(env, tmpdir):
    cache_dir = tmpdir / "cache"  # construct a path without the verbose `os.path.abspath` function
    cache_dir.ensure(dir=True)  # make sure it exists on the filesystem as a directory
    build_dir = tmpdir / "build"
    build_dir.ensure(dir=True)
    # ...
    assert not is_interesting(cache_dir / "another.file")  # much shorter
```